libtelemetry/src/main/java/com/mapbox/android/telemetry/FlushQueueCallback.java

```diff
@@ -3,5 +3,5 @@
 
 interface FlushQueueCallback {
```

Comment:
same here, i think this interface is redundant, should be put to bed as well.

Reply:
Yeah, I have the same thought when I read here